nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -87,6 +87,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [transfer-sh](https://github.com/dutchcoders/transfer.sh), a tool that supports easy and fast file sharing from the command-line. Available as [services.transfer-sh](#opt-services.transfer-sh.enable).
 
+- [AppImage](https://appimage.org/), a tool that supports running AppImages on NixOS. Available as [programs.appimage](#opt-programs.appimage.enable).
```

Comment:
This is a bit weird as it suggests this is somehow new and doesn't name the wrapper script (appimage-run). I'd rather announce the binfmt option; it's the star of the show I think.

Reply:
Does this look better now?

```markdown
- binfmt option for AppImage-run to support running [AppImage](https://appimage.org/)'s seamlessly on NixOS.. Available as [programs.appimage.binfmt](#opt-programs.appimage.binfmt).
```